Add new document to address working parameters (#35)
patcg/docs-and-reports: working-parameters/README.md

> # Working Parameters for Private Measurement Technical Specification MVP

Reviewer: Can we come up with a term other than "parameters" which invokes things like concrete epsilon levels, etc. Maybe "design dimensions" to align with the other doc?

Author: Sure, I'm fine with "design dimensions". I initially had "assumptions" written down, but that didn't feel quite right either.

Reviewer 2: Is it accurate to refer to them as "constraints"? That's what I consider them and it may be more recognizable, domain-specific term for folks reviewing the doc.

Author: In some cases, the result is a constraint, but it seems like "design dimension" fits a bit better (and those dimensions may or may not be constrained in specific ways.) Does that make sense?

At our September 2022 meeting at TPAC, 8 people signed up to be editors to draft a _Private Measurement Technical Specification MVP._  That group produced [Private Measurement Important Dimensions for Attribution.](https://github.com/patcg/docs-and-reports/tree/main/design-dimensions)

> At the February 2023 in person meeting in NYC, we ran a [survey and discussed the results](https://github.com/patcg/meetings/issues/91). We were able to discuss two of these topics in detail at the meeting, and intend to continue that conversation in upcoming meetings. As part of those discussions, we reached soft consensus on working parameters that the editors of the _Private Measurement Technical Specification MVP_ can work from.

Author: ```suggestion
At the February 2023 in person meeting in NYC, we ran a [survey and discussed the results](https://github.com/patcg/meetings/issues/91). We were able to discuss two of these topics in detail at the meeting, and intend to continue that conversation in upcoming meetings. As part of those discussions, we had general agreement on working parameters that the editors of the _Private Measurement Technical Specification MVP_ can use to make progress on the spec. 
```

This document serves to capture those parameters (and is intended to be updated as more are discussed and reach soft consensus.)

> # Parameters with Soft Consensus

Author: ```suggestion
# Parameters with General Agreement
```

## Server Architecture may exist

> The community group has reached soft consensus that the _Private Measurement Technical Specification MVP_ need not be limited to only on-device processing, however there is a very high bar for any server architecture that is included. This included both a high level of security (which the editors will align with the ongoing [Threat Model](https://github.com/patcg/docs-and-reports/tree/main/threat-model) work) as well as ability to explain the privacy and security properties of the system to end users.

Author: ```suggestion
The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ need not be limited to only on-device processing, however there is a very high bar for any server architecture that is included. This included both a high level of security (which the editors will align with the ongoing [Threat Model](https://github.com/patcg/docs-and-reports/tree/main/threat-model) work) as well as ability to explain the privacy and security properties of the system to end users.
```

## Privacy defined by Differential Privacy 

We’ve explored three main definitions of privacy:

1. Information theoretic
2. K-anonymity
3. Differential privacy

> The community group has reached soft consensus that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not rule out the use of other definitions of privacy being used in conjunction with differential privacy, however any proposal should aim to achieve a differential privacy guarantee.

Author: ```suggestion
The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not rule out the use of other definitions of privacy being used in conjunction with differential privacy, however any proposal should aim to achieve a differential privacy guarantee.
```

# Working Parameters for Private Measurement Technical Specification MVP

At our September 2022 meeting at TPAC, 8 people signed up to be editors to draft a _Private Measurement Technical Specification MVP._  That group produced [Private Measurement Important Dimensions for Attribution.](https://github.com/patcg/docs-and-reports/tree/main/design-dimensions)

At the February 2023 in person meeting in NYC, we ran a [survey and discussed the results](https://github.com/patcg/meetings/issues/91). We were able to discuss two of these topics in detail at the meeting, and intend to continue that conversation in upcoming meetings. As part of those discussions, we reached soft consensus on working parameters that the editors of the _Private Measurement Technical Specification MVP_ can work from.

This document serves to capture those parameters (and is intended to be updated as more are discussed and reach soft consensus.)

# Parameters with Soft Consensus

> ## Server Architecture may exists

Reviewer 3: ```suggestion
## Server Architecture may exist
```

> The community group has reached soft consensus that the _Private Measurement Technical Specification MVP_ need not be limited to only on-device processing, however there is a very high bar for any server architecture that is included. This included both a high level of security (which the editors will align with the ongoing [Threat Model](https://github.com/patcg/docs-and-reports/tree/main/threat-model) work) as well as ability to explain the privacy and security properties of the system to end users.

Reviewer 2: I think rather than "need not be limited to only on-device processing, however there is a very high bar for any server architecture that is included." say something like: "may include server-based dependencies so long as the server architecture meets minimum security and privacy guarantee requirements".

Author: ```suggestion
The community group has reached soft consensus that the _Private Measurement Technical Specification MVP_ may include server processing dependencies, however any server architechture must achieve both a high level of security (which the editors will align with the ongoing [Threat Model](https://github.com/patcg/docs-and-reports/tree/main/threat-model) work) as well as ability to explain the privacy and security properties of the system to end users.
```

How about this? I would prefer to avoid "minimum requirements", as it could imply that any solution which meets those requirements would be satisfactory. I think to Luke's point in the meeting, these requirements are necessary, but not always sufficient.

## Privacy defined by Differential Privacy 

We’ve explored three main definitions of privacy:

> 
> 

Reviewer 3: ```suggestion
```

1. Information theoretic
2. K-anonymity
3. Differential privacy

> The community group has reached soft consensus that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not rule out the use of other definitions of privacy being used in conjunction with differential privacy, however any proposal should aim to achieve a differential privacy guarantee.

Reviewer 2: Think the last sentence could be made clearer, suggest:

This does not preclude the use of other privacy definitions in conjunction with differential privacy, however any proposal should support differential privacy guarantees.

Author: ```suggestion
The community group has reached soft consensus that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not preclude the use of other privacy definitions in conjunction with differential privacy, however any proposal should aim to provide differential privacy guarantees.
```
Thanks @bmayd, good suggestion. Made one small tweak.
